test/gpdb/scripts/test_pitr_gpdb6.sh

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+PGBACKREST_TEST_DIR=/home/gpadmin/test_pgbackrest
+PGBACKREST_BIN=/usr/local/bin
+GPHOME=/usr/local/greenplum-db-devel
+
+# Starting up demo cluster
+source "$GPHOME/greenplum_path.sh"
+pushd gpdb_src/gpAux/gpdemo
+make create-demo-cluster WITH_MIRRORS=true
+source gpdemo-env.sh
+popd
+
+# Creating backup and log directories for pgbackrest
+TEST_NAME=$(basename "${0%.sh}")
+mkdir -p "$PGBACKREST_TEST_DIR/logs/$TEST_NAME"
+mkdir -p "$PGBACKREST_TEST_DIR/$TEST_NAME"
+DATADIR="${MASTER_DATA_DIRECTORY%*/*/*}"
+MASTER=${DATADIR}/qddir/demoDataDir-1
+PRIMARY1=${DATADIR}/dbfast1/demoDataDir0
+PRIMARY2=${DATADIR}/dbfast2/demoDataDir1
+PRIMARY3=${DATADIR}/dbfast3/demoDataDir2
+MIRROR1=${DATADIR}/dbfast_mirror1/demoDataDir0
+MIRROR2=${DATADIR}/dbfast_mirror2/demoDataDir1
+MIRROR3=${DATADIR}/dbfast_mirror3/demoDataDir2
+
+# Filling the pgbackrest.conf configuration file
+cat <<EOF > $PGBACKREST_TEST_DIR/pgbackrest.conf
```

Comment:
The standard configuration file name is `/etc/pgbackrest.conf`. We can create file with the standard name and do not use the `--config` option.

Reply:
/home/gpadmin/pgbackrest/test/gpdb/scripts/test_pitr_gpdb6.sh: line 27: /etc/pgbackrest.conf: Permission denied

Reply:
Is it possible to create `/etc/pgbackrest.conf`in Dockerfile, or use `sudo` or some other way?